actors/runtime/src/main/java/cloud/orbit/actors/runtime/MessageLoopback.java

```diff
@@ -75,7 +75,7 @@ public Task write(final HandlerContext ctx, final Object msg) throws Exception
                     {
                         clonedPayload = cloner.clone(originalPayload);
                     }
-                    catch (Exception e)
+                    catch (LinkageError | Exception e)
```

Comment:
other option is catching error and throwable, and rethrowing error immediately, wdyt?